website/docs/d/schematics_action.html.markdown

```diff
@@ -0,0 +1,205 @@
+---
+layout: "ibm"
```

Comment:
Add subcategory similar to altest docs in all doc files and remove sidebar_current

Reply:
Done.